pdufour/sfmovies
# feat (api) add api for movies db

# what
exposes endpoints to create movies

# testing instructions 
`curl -X POST -d '{"name": "foo"}' http://localhost:3000/movies -H 'Content-Type: application/json'` should create a new movie and return it
